website/app/components/doc/banner/index.js

```diff
@@ -1,6 +1,26 @@
 import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+
+export const TYPES = [
+  'info',
+  'information',
+  // 'success',
```

Comment:
Is there a reason for leaving this out?

Reply:
Yes, I expect sooner or later to be added :)